pyproject.toml

```diff
@@ -25,15 +25,15 @@ kgcl-schema = "^0.3.1"
 funowl = "^0.1.12"
 gilda = {version = "^0.10.1", optional = true}
 kgcl-rdflib = "^0.3.0"
-pystow = "^0.5.0"
+pystow = "^0.4.4"
 class-resolver = "^0.3.10"
 ontoportal-client = "0.0.3"
 bioregistry = "^0.5.64"
 prefixmaps = "^0.1.2"
 ols-client = "^0.1.1"
 pandas = "^1.5.1"
 linkml-renderer = "^0.1.2"
-airium = "^0.2.5"
```

Comment:
was there a reason for this?

Reply:
Reverted in 9e38b39c9106440256fe1ad5407bd3c4187d0338

Reply:
I don't know why that happened.